lib/ui/survey_question/answer/answer_nps.dart

```diff
@@ -25,6 +27,21 @@ class AnswerNps extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final selectedAnswerIndex = ref.watch(selectedNpsIndexProvider);
 
+    ref.listen(surveyNextQuestionsProvider, (_, displayType) {
+      if (displayType.value == DisplayType.nps) {
+        if (selectedAnswerIndex >= 0 && selectedAnswerIndex < answers.length) {
```

Comment:
For the kind of nps question, there is a default 0 answer in the survey detail response. We could use it as the default answer if no answer selected

![image](https://github.com/Wadeewee/survey-flutter-ic-pooh/assets/6950766/fcbe44ed-9ccc-46ce-afdd-fb65a5b899db)